omero/sysadmins/python3-migration.rst

```diff
@@ -0,0 +1,128 @@
+CentOS 7 and Ubuntu 18.04.
+
+Both 3.5 and 3.7 should work and are slated to have support added, but Python 3.6 has been
+the focus of testing during the migration.
+
+Similarly, other operating systems are slated for having support added, but help from the
+community would be very welcome! Obvious next candidates are CentOS 8 and Ubuntu 20.04.
+
+Debian is more of an issue. Debian 9 is still on Python 3.5 and Debian 10 has moved to Ice 3.7
+so neither is particularly well supported at the moment. We are examining workarounds like
+building Ice from source and/or building a .deb package.
+
+Other prerequisites
+-------------------
+
+OMERO's other prerequisites have not changed substantially but if you would like to take this
+opportunity to move to the :doc:`recommended version<version-requirements>` for all requirements,
+the current choices are:
+
+- Ice 3.6 (non-optional)
+- Java 11
+- Nginx 1.14 or higher
+- PostgreSQL 11
+
+Other options
+-------------
+
+The installation walkthroughs provided in the documentation try to stick to a minimum installation.
+The only requirements are an understanding of the Unix shell, the standard package manager for your
+platform, and the regular Python distribution mechanisms.
+
+However, more advanced installation mechanisms are available if you are interested and have familiarity
+with the given mechanism:
+
+- `Ansible roles <https://galaxy.ansible.com/ome>`_ are available for most installation steps. The primary
+  roles, `omero-server` and `omero-web` have not yet been released and will need to be installed from
+  GitHub.
+
+- A `conda channel <https://anaconda.org/ome>_` provides pre-built packages needed by OMERO if you prefer
+  to use Anaconda/Miniconda instead of the Python distribution provided by your platform.
+
+- `Docker images <https://hub.docker.com/u/openmicroscopy>`_ are also available. Both the `omero-server`
+  and `omero-web` images are considered production quality.
+
+Please get in touch at https://forum.image.sc/c/data if you have any questions.
+
+OMERO.server
+------------
+
+The steps for an OMERO.server installation have not changed substantially.
+
+Download the OMERO.server.zip as you would usually do, and unpack it under your
+installation directory. We suggest: file:`/opt/omero/server/OMERO.server`
+
+We highly recommend a virtualenv-based installation for all of the Python
+dependencies. Follow the :doc:`standard installation instructions <unix/server-installation>`
+for your platform.
+All instructions ...
+
+Once you have your installation in place, you will need to follow the
+:doc:`standard upgrade instructions <server-upgrade>`, working from
+a *copy* of your data.
+
+OMERO.web
+---------
+
+Although it is possible to also follow the previous installation steps
+for OMERO.web, installation no longer requires downloading a package from
+https://downloads.openmicroscopy.org. If you choose to follow this route,
```

Comment:
``if you choose to follow this route...`` - spell out which route you mean - the previous sentence mentions two different routes.